src/Domain/Fund.php

```diff
@@ -75,4 +89,54 @@ public function getCurrencyCode(): string
     {
         return $this->currencyCode;
     }
+
+    /**
+     * Uses database copies of all data, not the Redis `Matching\Adapter`. Intended for use
+     * after a campaign closes.
+     *
+     * @return array{totalAmount: numeric-string, usedAmount: numeric-string, currencyCode: string}
+     */
+    public function getAmounts(): array
+    {
+        $totalAmount = Money::fromPoundsGBP(0);
+        $usedAmount = Money::fromPoundsGBP(0);
+
+        foreach ($this->campaignFundings as $campaignFunding) {
```

Comment:
We can only add currency codes if they're equal. IMHO it might be worth making a decision to properly remove multi-currency from matchbot at some point if we don't see any likelyhood of using a different currency, but while its there we can add this

```suggestion
        foreach ($this->campaignFundings as $campaignFunding) {
        Assertion::eq($campaignFunding->getCurrencyCode(), $this->currencyCode);
        
```